components/cli/cmd/cellery/test.go

```diff
@@ -77,7 +77,7 @@ func newTestCommand(cli cli.Cli) *cobra.Command {
 			return nil
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			commands.RunTest(cli, args[0], name, startDependencies, shareAllInstances, dependencyLinks, envVars,
+			image2.RunTest(cli, args[0], name, startDependencies, shareAllInstances, dependencyLinks, envVars,
```

Comment:
No errors returned from RunTest method?

Reply:
test command is not refactored yet, therefore RunTest still does not return any error.